### PR TITLE
Updating to work with React 0.14.0-rc1

### PR DESCRIPTION
**Pull request by nickpresta**
* Updated peerDependencies
* Updated render method to not mutate props
* Removed bit about babel polyfill

### CODE REVIEW

**Comment by nickpresta** [comment]
This should fix https://github.com/andreypopp/react-textarea-autosize/issues/47 and https://github.com/andreypopp/react-textarea-autosize/issues/53

**Comment by tomatau** [comment]
I've forked this as well and am tracking built lib so that can install directly from github repo. Also working in 0.14-rc1
```json
"dependencies": {
"tomatau/react-textarea-autosize#react-0.14",
```

File: src/TextareaAutosize.js [hunk 1]
```diff
@@ -68,22 +68,22 @@ export default class TextareaAutosize extends React.Component {
   }
 
   render() {
-    let {valueLink, onChange, ...props} = this.props;
-    props = {...props};
+    let {valueLink, onChange} = this.props;
+    let mergedProps = {...this.props};
```
Comment (andreypopp) [inline src/TextareaAutosize.js]: Why this change?

**Comment by andreypopp** [comment]
Closing in favor of #44 which was submitted first and appears to be more targeted towards a single change (updating codebase for react 0.14).

**Comment by nickpresta** [comment]
@andreypopp if you don't update the props to make a copy and use that, you get errors about mutating props (which was discouraged in React 0.13):

```
ERROR: 'Warning: `textarea` was passed a style object that has previously been mutated. Mutating `style` is deprecated. Consider cloning it beforehand. Check the `render` of `TextareaAutosize`. Previous style: {"height":null}. Mutated style: {"height":null}.'
```

**Comment by andreypopp** [comment]
@nickpresta that's interesting... at kind 72 code already clones `props` object.
